lumicks/pylake/detail/utilities.py

```diff
@@ -0,0 +1,17 @@
+
+def first(iterable, condition=lambda x: True):
+    """
+    Returns the first item in the `iterable` that satisfies the `condition`.
```

Comment:
The first line of the docstring should be inline with the triple quotes. Also, it should be imperative: "Return" instead of "Returns".